### ISSUE TITLE
Adding the Abhijit meeting notes into manuscript of article 01

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#278]
from Abhijit meeting at 04/12/2023 there were some additional has been gathered that can be added as useful information into manuscript

**Comment by hmehryar** [comment]
Abhijit questions
How the loss function is working
What is the loss/validation loss function formula
What is the accuracy/validation accuracy formula

**Mentioned in chap3** 
> in the images the coordinates are
>  etha: repidity
>  phi: athymothal angel
> p_t: weight
> events are pb-pb (lead-lead) simulations
> we are simulating mater-lbt events with the static medium
> 

**Mentioned in chap6**
> for the rest of research 
> predicting alpha_s value
> predicting q_0 value
> making a dataset with the same params with a hydro profile in matter-lbt 
> get the data from yasokee/chathuranga/amit
